tsnet/tsnet.go

```diff
@@ -357,9 +366,22 @@ func (s *Server) start() error {
 			logf("localapi serve error: %v", err)
 		}
 	}()
+	closePool.add(s.localAPIListener)
```

Comment:
this one's kinda pointless?

Reply:
The set of closed resources matches what occurs in `Server.Shutdown`. It's probably worth keeping so that there's correlation between the two.